From 744442fa48d3c8ade4bfbda4283bb0a18addc0cf Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 18 Jan 2024 15:25:29 +0100
Subject: [PATCH] com.webos.service.configurator.perm.json: Update the ACG
 group names

Fixes various issues:

WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: Found 46 group(s) used in client-permissions.d but not defined
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: === LIST BEGIN ===
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'activities.manage' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.configurator.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'applications.query' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.configurator.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   org.webosports.service.audio.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'configurator.callbacks' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.configurator.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'database.internal' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.configurator.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   org.webosports.app.memos.app.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'devices' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.configurator.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   org.webosports.service.audio.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'ppm.private' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.configurator.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'settings' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.app.mediagallery.app.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.configurator.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   org.webosports.service.audio.perm.json
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: 'tts.internal' being used in:
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   com.webos.service.configurator.perm.json

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending

 .../com.webos.service.configurator.perm.json   | 18 ++++++++++--------
 1 file changed, 10 insertions(+), 8 deletions(-)

diff --git a/files/sysbus/com.webos.service.configurator.perm.json b/files/sysbus/com.webos.service.configurator.perm.json
index 27289ca..f6180aa 100644
--- a/files/sysbus/com.webos.service.configurator.perm.json
+++ b/files/sysbus/com.webos.service.configurator.perm.json
@@ -3,7 +3,6 @@
         "activity.operation",
         "database.operation",
         "filecache.management",
-        "configurator.callbacks",
         "tempdatabase.operation",
         "epgdatabase.operation",
         "keymgrdatabase.operation",
@@ -13,14 +12,16 @@
         "tempdatabase.management",
         "mediadatabase.management",
         "database.management",
-        "database.internal",
-        "activities.manage",
-        "settings",
-        "devices",
+        "database.operation",
+        "preferences.applicationpropertyquery",
+        "preferences.systempropertyquery",
+        "physicaldevice.management",
+        "physicaldevice.query",
         "networking.query",
-        "applications.query",
-        "tts.internal",
-        "ppm.private",
+        "application.launcher",
+        "application.operation",
+        "tts.management",
+        "tts.operation",
         "application.launcher",
         "calendarremindersservice.operation",
         "contactslinkerservice.operation",
